src/style.rs

```diff
@@ -266,6 +266,8 @@ pub enum Dimension {
     Points(f32),
     /// The dimension is stored in percentage relative to the parent item.
     Percent(f32),
+    /// The dimension is stored as the sum of percentage relative to the parent item and point.
```

Comment:
I think we should add a little bit more color to the docs here, explaining why this is useful (and when you might want to use it).